[FireEye HX] fetch-incidents command did not fetch more than 50 incidents (#24663)
demisto/content: Packs/FireEyeHX/ReleaseNotes/2_3_2.md

#### Integrations
##### FireEye Endpoint Security (HX) v2

> - Fixed an issue where ***fetch-incidents*** command did not fetch more than 50 incidents.

Reviewer: ```suggestion
- Fixed an issue where ***fetch-incidents*** command would not fetch more than 50 incidents at a time.
```

> - Fixed an issue where ***fetch-incidents*** command would not fetch more than 50 incidents at a time.

Reviewer 2: ```suggestion
- Fixed an issue where the ***fetch-incidents*** command would not fetch more than 50 incidents at a time.
```

> - Fixed an issue where the ***fireeye-hx-file-acquisition*** command failed due timeout, from now the timeout is 30 minutes.

Reviewer: ```suggestion
- Fixed an issue where the ***fireeye-hx-file-acquisition*** command timed out after 10 minutes, by updating the limit to 30 minutes.
```

> - Fixed an issue where the ***fireeye-hx-file-acquisition*** command failed due timeout, from now the timeout is 30 minutes (previously 10 minutes).

Reviewer 2: ```suggestion
- Fixed an issue where the ***fireeye-hx-file-acquisition*** command failed due to a timeout. Updated the timeout to 30 minutes (previously it was 10 minutes).
```
